libs/blocks/global-navigation/global-navigation.js

```diff
@@ -481,6 +483,11 @@ class Gnav {
       return children;
     };
 
+    const onAnalyticsEvent = (data) => {
+      // TODO: update based on requirements
+      console.log('analytics data', data);
```

Comment:
I had already pressed the comment button for the `console` message, but then I saw the code comment 😅 Since this is going to the `gnav` branch, it should be fine as-is for now